bcgov/cas-registration
# Giraffe/task/190 monorepo frontend testing

Addresses bcgov/cas-reporting#190 - [Project link](https://github.com/orgs/bcgov/projects/123/views/1?pane=issue&itemId=57275284)
Adds Vitest shared configuration to monorepo and configs into `shared/components` lib and `reporting` app.

## Changes 🚧

- Added Vite/Vitest via @nx packages
- Added library with shared global setup for Vitest
- Added vitest config to shared components and reporting app
- Added Github workflow to run Nx tests

## To test 🔬

> You may need to run `yarn` in the `./bciers` directory to install and update packages.
- Navigate to `./bciers` and run (if nx is not installed globally, first `npx `)`nx run-many -t test --skipNxCache`. It should run tests across two Nx projects. 
- You can also run `nx test reporting` or `nx test components` to test the individual Nx projects.
